qiita_db/commands.py

```diff
@@ -300,3 +304,101 @@ def load_parameters_from_cmd(name, fp, table):
                          "The format is PARAMETER_NAME<tab>VALUE")
 
     return constructor.create(name, **params)
+
+
+def update_preprocessed_data_from_cmd(sl_out_dir, study_id):
```

Comment:
Is it possible to use the queue system in this function to ensure that the operation is atomic? Otherwise, I fear we may run into integrity issues